### PR TITLE
fix(missing user): changed logic to get users

### LINKED ISSUE #31: User not available when configuring default mount point settings
## Describe the bug
I installed the extension and now I need to set `Default mount point settings`. I am using Ubuntu WSL but no User is shown.

## Screenshots

image

### PR DESCRIPTION
**Pull request by Pive01**
Closes #31

### CODE REVIEW

File: ui/src/services/util/format.ts [hunk 1]
```diff
@@ -6,21 +6,24 @@ export function removeNullBytes(str: string): string {
...
+    .filter(distro => !EXCLUDED_OS_WSL.includes(distro));
 }
 
+const EXCLUDED_USER_WSL = ['[.]', '[..]', '\r'];
+
 export function getUsersFromBinaryWindows(res: string): string[] {
-  return res.split('\n').slice(5, -2) // get only directories rows
-    .map(str => str.split(' ').at(-1) // get only dir name (ex "luca\r")
-      .slice(0, -1)); // remove newline 
+  return res.split('\n').filter(string => string.startsWith('[')) // get only direcotires row (they are in form "[user1] [user2]" )
```
Comment (whummer) [inline ui/src/services/util/format.ts:23]:
```suggestion
  return res.split('\n').filter(string => string.startsWith('[')) // get only directory rows (they are in form "[user1] [user2]" )
```

Comment (whummer) [reply ui/src/services/util/format.ts:23]: Also, I think it would be good to apply a `trim()` to the string, to remove potential whitespaces..

**Review by whummer — APPROVED** [review]
LGTM! 👍 Thanks for the fix @Pive01 . Unable to test this myself (no access to Windows machine right now), but based on the bug and the PR description it seems to me that should work.

**Comment by valeryan** [comment]
Hope this works, I have been unable to use the localstack extension because it can't find my user.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**